src/jit/codegenxarch.cpp

```diff
@@ -2651,9 +2638,14 @@ CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
         getStructReturnRegisters(type0, type1, &retReg0, &retReg1);
 
         assert(retReg0 != REG_NA && retReg1 != REG_NA);
-
+        
+        // Copy of a struct to a well typed location on the stack. No need to call the 
+        // GC-ness of each slot. Need to make the store of the two slots
+        // atomic for the purposes of GC.
+        getEmitter()->emitDisableGC();
```

Comment:
This comment doesn't make sense to me, and is not consistent with the code below. If you are disabling GC, then the stores do not need to be atomic. I suspect that what you mean is that the stores must be explicit so that the GC range info will be updated appropriately by the emitter. Also, what does "call the GC-ness of each slot" mean? Are you saying that you don't need to call a helper?